packages/neovim/src/utils/findNvim.ts

```diff
@@ -118,14 +122,31 @@ function compareVersions(a: string, b: string): number {
  *
  * @param opt.minVersion See {@link FindNvimOptions.minVersion}
  * @param opt.orderBy See {@link FindNvimOptions.orderBy}
+ * @param opt.stopOnFirstMatch See {@link FindNvimOptions.stopOnFirstMatch}
  */
-export function findNvim(opt: FindNvimOptions = {}): Readonly<FindNvimResult> {
+export function findNvim({
+  minVersion,
+  orderBy,
+  stopOnFirstMatch,
+}: FindNvimOptions = {}): Readonly<FindNvimResult> {
   const paths = process.env.PATH?.split(delimiter) ?? [];
-  const pathLength = paths.length;
+  paths.push(
+    '/usr/local/bin',
+    '/usr/bin',
+    '/opt/homebrew/bin',
+    '/home/linuxbrew/.linuxbrew/bin',
+    '/snap/nvim/current/usr/bin'
+  );
+  const home = process.env.HOME;
+  if (home) {
+    paths.push(`${home}/bin`, `${home}/.linuxbrew/bin`);
+  }
+
   const matches = new Array<NvimVersion>();
   const invalid = new Array<NvimVersion>();
-  for (let i = 0; i !== pathLength; i = i + 1) {
-    const nvimPath = join(paths[i], windows ? 'nvim.exe' : 'nvim');
+  const uniquePaths = new Set(paths);
```

Comment:
TODO: on Windows, this Set won't help much (need to normalize the slashes and perhaps drive letters).

Reply:
Fixed